manifests/l/LutzRoeder/Netron/6.2.0/LutzRoeder.Netron.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+PackageIdentifier: LutzRoeder.Netron
+PackageVersion: 6.2.0
+PackageName: Netron
+PackageLocale: en-US
+PackageUrl: https://github.com/lutzroeder/netron
+Publisher: Lutz Roeder
+PublisherUrl: https://github.com/lutzroeder/netron
+PublisherSupportUrl: https://github.com/lutzroeder/netron/issues
+Author: Lutz Roeder
+License: Copyright (c) Lutz Roeder
```

```suggestion
License: MIT
```
As per the [manifest locale specification](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.2.0/locale.md#fields), a copyright is not considered a license.

> This key represents the license governing the use and or distribution for the product. This could be an open source license, or a commercial license. Please note that a copyright is not considered a license. If there is no available information on a product's license, "Proprietary" should be the value in this field.